pandas/core/internals/blocks.py

```diff
@@ -925,26 +925,28 @@ def putmask(
         inplace: bool = False,
         axis: int = 0,
         transpose: bool = False,
-    ):
+    ) -> List["Block"]:
         """
         putmask the data to the block; it is possible that we may create a
         new dtype of block
 
-        return the resulting block(s)
+        Return the resulting blocks.
```

Comment:
can return a list with a single block? if so, i _think_ `block(s)` is more appropriate